android/app/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+<uses-permission android:name="android.permission.RECEIVE_SMS"/>
```

Comment:
use Google's SMS Retriever API, you dont need this permssion to enter otp.
I want to keep minimal permission for our app.